packages/browser/src/lib/client-hints/index.ts

```diff
@@ -0,0 +1,25 @@
+export async function clientHints(
+  hints?: string[]
+): Promise<UADataValues | undefined> {
+  const userAgentData = ((): NavigatorUAData | undefined =>
+    (navigator as any).userAgentData)()
+
+  if (!userAgentData) return
+
+  let userAgentDataContext: UADataValues = {
+    brands: userAgentData.brands,
+    mobile: userAgentData.mobile,
+    platform: userAgentData.platform,
+  }
```

Comment:
Rather than constructing this object, I think an idiomatic thing is to just do const lowEntropyValues = userAgentData.toJSON() -- I updated my code sample below.